### PR TITLE
Update dpkg package

### PR DESCRIPTION
**Pull request by JL2210**
Update dpkg to 1.19.1
Tested on x86_64
Version 1.19.1 of dpkg resolves po errors of previous version.
Safe for pre-compiled binaries.

### CODE REVIEW

File: packages/dpkg.rb [hunk 1]
```diff
@@ -3,20 +3,20 @@
 class Dpkg < Package
   description 'A medium-level package manager for Debian'
   homepage 'https://anonscm.debian.org/git/dpkg/'
-  version '1.18.25'
-  source_url 'https://salsa.debian.org/dpkg-team/dpkg/-/archive/1.18.25/dpkg-1.18.25.tar.gz'
-  source_sha256 '93470770161bb15fd7988da1890aecda322e19c3fc858d6cb8f33417e5fe01e9'
+  version '1.19.1'
+  source_url 'http://http.debian.net/debian/pool/main/d/dpkg/dpkg_1.19.1.tar.xz'
```
Comment (uberhacker) [inline packages/dpkg.rb]: What is wrong with `https://salsa.debian.org/dpkg-team/dpkg/-/archive/1.19.1/dpkg-1.19.1.tar.gz`?

File: packages/dpkg.rb [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
 
   def self.build
-    system "git clone https://salsa.debian.org/dpkg-team/dpkg.git -b 1.18.25"
-    Dir.chdir ("dpkg") do
-      system "autoreconf -i -f"
-      system "./configure --libdir=#{CREW_LIB_PREFIX} --prefix=#{CREW_PREFIX}"
-      system "make"
-    end
+    system "/usr/bin/env",
+           "PERL_LIBDIR=#{CREW_PREFIX}/lib/perl5/site_perl/",
```
Comment (uberhacker) [inline packages/dpkg.rb]: Can you please clue me in why this is needed?

File: packages/dpkg.rb [hunk 3]
```diff
@@ -3,20 +3,20 @@
...
   homepage 'https://anonscm.debian.org/git/dpkg/'
-  version '1.18.25'
-  source_url 'https://salsa.debian.org/dpkg-team/dpkg/-/archive/1.18.25/dpkg-1.18.25.tar.gz'
-  source_sha256 '93470770161bb15fd7988da1890aecda322e19c3fc858d6cb8f33417e5fe01e9'
+  version '1.19.1'
+  source_url 'http://http.debian.net/debian/pool/main/d/dpkg/dpkg_1.19.1.tar.xz'
+  source_sha256 'ae3978a6b7bddc3e3196804ae0d49ea008c84a8a7a60b7d212af1e1d469e7ccf'
 
   depends_on 'bz2'
   depends_on 'xzutils'
```
Comment (uberhacker) [inline packages/dpkg.rb]: `depends_on 'perl'`?

Comment (uberhacker) [reply packages/dpkg.rb]: Actually, this is in "core" so nevermind.

File: packages/dpkg.rb (hunk 1 above)
Comment (JL2210) [reply packages/dpkg.rb]: REmember the fact that we had tio use git before? This source tarball has a configure script.

File: packages/dpkg.rb (hunk 2 above)
Comment (JL2210) [reply packages/dpkg.rb]: `PERL_LIBDIR`? That's where dpkg installs its perl module files. They are needed for `dpkg-architecture` to work. If you're talking about `/usr/bin/env`, try doing it without. It fails with `bash: PERL_LIBDIR=/usr/local/lib/perl5/site_perl/: command not found`

**Pull request merged**